pkg/ioutils/fswriters_linux.go

```diff
@@ -0,0 +1,11 @@
+package ioutils
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+func fdatasync(f *os.File) error {
+	return unix.Fdatasync(int(f.Fd()))
```

Comment:
fdatasync(3p) says this is in POSIX, but in golang.org/x/sys/unix, only Linux, Solaris, and AIX have a corresponding function.  I don't think we target Solaris or AIX, so this should be fine.